lib/doorkeeper/models/access_grant_mixin.rb

```diff
@@ -24,6 +24,10 @@ module AccessGrantMixin
       validates :token, uniqueness: true
 
       before_validation :generate_token, on: :create
+
+      def uses_pkce?
+        code_challenge.present? && code_challenge_method.present?
```

Comment:
ostensibly, `code_challenge.present?` would be enough to enable PKCE for the spec.  If we plan on open sourcing this change, then I think we should reflect that